core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/ssp_config.h

```diff
@@ -25,9 +25,10 @@
 
 // On Linux, prefer to use getrandom, though it isn't available in
 // GLIBC before 2.25.
-#if defined(__linux__)                       \
-    && (!defined(__GLIBC__) || __GLIBC__ > 2 \
-        || (__GLIBC__ == 2 && __GLIBC_MINOR__ >= 25))
+#if defined(__linux__)                              \
+    || defined(ESP_PLATFORM)                        \
```

Comment:
Should be `(defined(__linux__) || defined(ESP_PLATFORM))`

Reply:
OK.